src/NuGet.Services.EndToEnd/Support/TestSettings.cs

```diff
@@ -8,6 +8,20 @@ namespace NuGet.Services.EndToEnd.Support
 {
     public class TestSettings
     {
+        public enum Mode
+        {
+            EnvironmentVariables,
```

Comment:
I thought of this but decided against because local is confusing since that could suggest localhost gallery or search.